website/www/site/content/en/documentation/sdks/python.md

```diff
@@ -46,7 +46,13 @@ new I/O connectors. See the [Developing I/O connectors overview](/documentation/
 for information about developing new I/O connectors and links to
 language-specific implementation guidance.
 
-## Using Beam Python SDK in your ML pipelines
+## Making machine learning inferences with Python
+
+To integrate machine learning models into your pipelines for making inferences, use the RunInference API for PyTorch and Scikit-learn models. If you are using TensorFlow models, you can make use of the
+[library from `tfx_bsl`](https://github.com/tensorflow/tfx-bsl/tree/master/tfx_bsl/beam). Further integrations for TensorFlow are planned. For more information, see [Beam issue #21442](https://github.com/apache/beam/issues/21442).
```

Comment:
You could say --- Further integrations for popular machine learning frameworks like Tensorflow are planned.

But I'm wondering if its worth saying anything at all, since this documentation could end up out of date soon.